public/assets/i18n/en.json

```
@@ -15,8 +15,28 @@
         }
       }
     },
+    "new-forum":{
+      "title": "Create Forum",
+      "form": {
+        "inputs": {
+          "title": {
+            "label": "Title",
+            "placeholder": "A forum"
+          },
+          "description": {
+            "label": "Description",
+            "placeholder": "This forum contains..."
+          },
+          "submit": {
+            "name": "Create"
```

Why name? Would label or text be more accurate?
```suggestion
            "label": "Create"
```